.github/workflows/ubuntu_test_cb.yaml

```diff
@@ -0,0 +1,33 @@
+name: build-and-test-compiler-base-ubuntu
```

Comment:
remove `ubuntu` in the name and yaml file name.

Reply:
fixed